hotshot-query-service/src/data_source/update.rs

```diff
@@ -139,16 +142,13 @@ where
                     Some(VidDisperseShare::V0(share)) => (
                         Some(VidCommonQueryData::new(
                             leaf.block_header().clone(),
-                            share.common.clone(),
+                            Some(share.common.clone()),
                         )),
-                        Some(share.share.clone()),
+                        Some(VidShare::V0(share.share.clone())),
                     ),
                     Some(VidDisperseShare::V1(share)) => (
-                        Some(VidCommonQueryData::new(
-                            leaf.block_header().clone(),
-                            share.common.clone(),
-                        )),
-                        Some(share.share.clone()),
+                        Some(VidCommonQueryData::new(leaf.block_header().clone(), None)),
```

Comment:
so new VID variant does not have common(). We do store common() data, which would be None in this case. I don't know how this is being used but need to think about this.